### PR TITLE
Initial release notes for 4.13.2

### PR DESCRIPTION
**Pull request by kcooney**

### CODE REVIEW

File: doc/ReleaseNotes4.13.2.md
Comment (marcphilipp) [inline doc/ReleaseNotes4.13.2.md:13]:
```suggestion
change explicitly destroyed the `ThreadGroup` that was created for the time-limited test. Numerous
people reported problems that were caused by explicitly destroying the `ThreadGroup`.

In this change, the code was updated to call  `ThreadGroup.setDaemon(true)` instead of destroying the
ThreadGroup.
```

**Review by marcphilipp — CHANGES_REQUESTED** [review]

**Comment by kcooney** [comment]
I accepted the changes. Friendly reminder to squash and merge this change when everyone is happy with it.

**Review by marcphilipp — APPROVED** [review]
Thanks! 👍

**Comment by kcooney** [comment]
I'm going to submit with just the one approval. I figure we'll have other opportunities to review the release notes before 4.13.2 is released.

**Comment by stefanbirkner** [comment]
IIRC for a PR of a JUnit maintainer one approval is enough

**Pull request merged**
